parser/build.gradle

```diff
@@ -83,7 +83,7 @@ build.dependsOn shadowJar
 task testParsing(dependsOn: ':test-app:assembleDebugAndroidTest', type: JavaExec) {
     classpath sourceSets.main.runtimeClasspath
     main = 'com.linkedin.dex.parser.DexParser'
-    args "${rootProject.project('test-app').buildDir}/outputs/apk/test-app-debug-androidTest.apk", "$buildDir"
+    args "${rootProject.project('test-app').buildDir}/outputs/apk/androidTest/debug/test-app-debug-androidTest.apk", "$buildDir"
```

Comment:
just double checking - this is a result of the bumps in AGP/Gradle moving the default output directory correct?

Reply:
yep, the new Android Gradle Plugin generates android test apk in `androidTest/debug` directory